source/uwp/Renderer/lib/MediaHelpers.cpp

```diff
@@ -316,12 +317,13 @@ HRESULT HandleMediaClick(
     HSTRING mimeType,
     IAdaptiveMediaEventInvoker* mediaInvoker)
 {
+    ComPtr<IMediaElement> localMediaElement{ mediaElement };
+
     // When the user clicks: hide the poster, show the media element, open and play the media
```

Comment:
Did this need to move? The access to the variable remains in the same scope as before.

Reply:
Moved it back.

---
In reply to: [202800202](https://github.com/Microsoft/AdaptiveCards/pull/1713#discussion_r202800202) [](ancestors = 202800202)